Lab 12

zilinbian56 opened the pull request:
Standard deep learning and neural networks have given astounding performances on many tasks, but they commonly do not capture model uncertainty. In some cases of prediction, however, it is quite important to tell how confident we are about the model estimation, not just the the prediction result. To achieve this goal, we utilize Bayesian neural networks to get a predictive distribution of results instead of point estimation in this section. Specifically, we walk through the process of enabling dropout and getting the estimation of the predictive distribution. Then, we perform experiments using Bayesian neural networks with different non-linearities.
--Lab12 (23 Apr 2019) by Zilin Bian, Di Sha. Edited by Jong Yeob Kim

zilinbian56 committed:
Updates from Overleaf

Atcold commented:
Plz, edit the first message, similarly to how it's done in #1.
Also, make the summary more generic, explaining what has been covered in the lesson and why, rather than listing the chapters that this PR is introducing.

zilinbian56 commented:
> Plz, edit the first message, similarly to how it's done in #1.
> Also, make the summary more generic, explaining what has been covered in the lesson and why, rather than listing the chapters that this PR is introducing.

Hi there,
We've changed the summary and updated it in the first message, please have a look at it.

Atcold commented:
If you check #1's first post, you'll see something like:

> -- Second part of Lecture 1 (28 Jan 2019) by Erica Dominic and Hongyu (Florence) Lu. Edited by Michael Gold

Add this missing line to your first post. Thanks.

Atcold committed:
Merge branch 'master' into master

Atcold commented on labs/12/theory.tex:
Replace this image with the code itself. Thanks.
See other PRs for how this has been done, with the minted package.

zilinbian56 replied:
Please check PR Lab12 #53

Atcold commented on labs/12/theory.tex:
`visual_caption` is already defined here https://github.com/Atcold/SP19-DL-collaborative-notes/blob/master/labs/07/theory.tex#L14

zilinbian56 replied:
Please check PR Lab12 #53
